pkg/registry/utils/count/nse_client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package count
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"google.golang.org/grpc"
+
+	"github.com/networkservicemesh/api/pkg/api/registry"
+
+	"github.com/networkservicemesh/sdk/pkg/registry/core/next"
+)
+
+// NSEClient is a client type for counting Register / Unregister / Find
+type NSEClient struct {
+	totalRegisterCalls, totalUnregisterCalls, totalFindCalls int32
+	registers, unregisters, finds                            map[string]int32
+	mu                                                       sync.Mutex
```

Comment:
These fields are not used. Please don't add dead code.

```suggestion
```

Reply:
Done